README.md

```diff
@@ -2,6 +2,220 @@
 
 [![Build Status](https://travis-ci.org/revdotcom/revai-java-sdk.svg?branch=develop)](https://travis-ci.org/revdotcom/revai-java-sdk)
 
+## Documentation
+
+See the [API docs](https://www.ai.rev.ai/docs) for more information about the API.
+
+## Install the SDK
+The recommended way to use the Rev.ai Java SDK is to import it into the project using Maven.
+
+      <dependency>
+        <groupId>ai.rev</groupId>
+        <artifactId>revai-java-sdk</artifactId>
+        <version>1.0.0</version>
+      </dependency>
+    
+## Installing it locally
+Once you've cloned the repo you can use Maven to build it locally and install it your local Maven .m2 repository.
+
+    mvn install -Dmaven.test.skip=true
+
+## Support
+
+We support Java 8 and 11.
+
+## Usage
+
+All you need to get started is your Access Token, which can be generated on
+your [Settings Page](https://www.ai.rev.ai/access_token). Create a client with the
+given Access Token:
+
+```
+// Initialize your client with your ai.rev access token
+String accessToken = "Your Access Token";
+ApiClient apiClient = new ApiClient(accessToken);
+```
+
+### Checking credits remaining
+
+```
+RevAiAccount revAiAccount = apiClient.getAccount(); 
+```
+
+### Submitting a job
+
+You can submit a local file
+```
+RevAiJob revAiJob = apiClient.submitJobLocalFile("./path/to/file.mp3");
+```
+
+or submit via a public url
+```
+RevAiJob revAiJob = apiClien.submitJobUrl("https://www.ai.ai.rev.ai/FTC_Sample_1.mp3");
+```
+
+or from FileInputStream, the filename is optional.
+```
+File file = new File("./path/to/file.mp3");
+FileInputStream fileInputStream;
+try {
+  fileInputStream = new FileInputStream(file);
+} catch (FileNotFoundException e) {
+  throw new RuntimeException("Could not find file [" + file.getName() + "]");
+}
+RevAiJob revAiJob = apiClient.submitJobLocalFile(fileInputStream, String fileName, RevAiJobOptions options);
+```
+
+`RevAiJob` objects contain all the information normally found in a successful response from our
+[Submit Job](https://www.ai.rev.ai/docs#operation/SubmitTranscriptionJob) endpoint.
+
+If you want to get fancy, all send job methods can take a `RevAiJobOptions` object which contains methods 
```

Comment:
Use consistent terminology:
```suggestion
If you want to get fancy, all submit job methods can take a `RevAiJobOptions` object which contains methods 
```

Reply:
I would also mention the list is non-exhaustive or reference a list of job options or the job options object definition instead of listing them out. We don't want to have to update this README's list every time we add a new job option.